Xamarin.Forms.Core.Android.UITests/Xamarin.Forms.Core.Android.UITests.csproj

```diff
@@ -59,7 +59,7 @@
       <HintPath>..\packages\NUnit.2.6.4\lib\nunit.framework.dll</HintPath>
     </Reference>
     <Reference Include="Xamarin.UITest">
-      <HintPath>..\packages\Xamarin.UITest.2.2.4\lib\net45\Xamarin.UITest.dll</HintPath>
+      <HintPath>..\packages\Xamarin.UITest.2.2.5\lib\net45\Xamarin.UITest.dll</HintPath>
```

Comment:
Why not to use PackageReference?

Reply:
Aslında referanslarım var ama dikkat etmemişsiniz sanırım.

Reply:
I mean use PackageReference instead of modifing both packages.config and path to references

Reply:
We will, but you can make a PR to help us out with that :P 

Thanks for the tip